### DISCUSSION TITLE
Edited Gcode not sent

### DISCUSSION THREAD

**Discussion by lagnat** [winder/Universal-G-Code-Sender#1926] [Help]
I cannot figure out if this is a bug or I'm doing something wrong.  If I load a file and send the Gcode, everything is fine.  If I make some edits to the Gcode after that and press Send, it sends the original code not my changes.  I've tried saving, loading, even closing the source tab, but Send always sends the original Gcode.  I have to restart UGS in order to send another file.  What am I missing?

UGS 2.0.11

**Comment by breiler** [comment]
What operating system are you using?
Can you attach your log file (I will assume you are running Windows): 
`C:/Users/[your username>]/.ugsplatform/2.X/dev/var/log/messages.log`
